site_forecast_app/app.py

```diff
@@ -129,7 +129,7 @@ def save_forecast(
     log.info(f"Saving forecast for site_id={forecast['meta']['site_uuid']}...")
 
     forecast_meta = {
-        "site_uuid": forecast["meta"]["site_uuid"],
+        "location_uuid": forecast["meta"]["site_uuid"],
```

Comment:
Is this dictionary key still correct?

Reply:
yea, it is, but it is confusing, I think Ill change them all to "location_uuid"